Improve CommonJS import from README

vkarpov15 opened the pull request:
AFAIK `require('mitt')` won't work by itself, you need to do `require('mitt').default`.

Thanks for the great lib :+1:

developit commented:
@vkarpov15 that shouldn't be the case - you can see the commonjs version (default version) has no `.default` here:
https://unpkg.com/mitt@1.1.3/dist/mitt.js

vkarpov15 commented:
Weird, I just tried it again and `const mitt = require('mitt')` worked perfectly. Thanks for taking the time to look at this and sorry for the false alarm.

```
$ node -v
v8.9.4
$ npm -v
5.6.0
$ npm install mitt

+ mitt@1.1.3
added 1 package and removed 1 package in 8.356s
$ node
> const mitt = require('mitt')
undefined
> mitt
[Function: n]
> mitt()
{ on: [Function: on],
  off: [Function: off],
  emit: [Function: emit] }
> 
```
